client/components/marketing-survey/gsuite-cancel-purchase-dialog/index.jsx

```diff
@@ -0,0 +1,201 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize, moment } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import * as steps from './steps';
+import Dialog from 'components/dialog';
+import enrichedSurveyData from 'components/marketing-survey/cancel-purchase-form/enriched-survey-data';
+import GSuiteCancellationFeatures from './gsuite-cancellation-features';
+import GSuiteCancellationSurvey from './gsuite-cancellation-survey';
+import notices from 'notices';
+import { recordTracksEvent } from 'state/analytics/actions';
+import wpcom from 'lib/wp';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class GSuiteCancelPurchaseDialog extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = this.initialState;
+	}
+
+	get initialState() {
```

Comment:
What is this `get` keyword? That's the first time I see it. Shouldn't we use `state={}` instead?

Reply:
It is a [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get). The problem I am solving is that I want to set the state to the "initial state" described by this function in multiple places: when the component is created, and when it resets because of a cancelling out of the form.

Reply:
Interesting. What are the benefits of using a getter instead of simply calling this function `getInitialState()`?

Reply:
really semantic sugar seems to be the major difference. you would use it like a property instead of a function i.e. `this.initialState` vs `this.initialState()`